src/main/drivers/pwm_output.c

```diff
@@ -245,11 +245,16 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
         idlePulse = 0;
         break;
 #ifdef USE_DSHOT
+	case PWM_TYPE_PROSHOT1000:
```

Comment:
Please align statements. Betaflight style is spaces, not tabs and indentation of 4.